gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/PipelineController.groovy

```diff
@@ -240,7 +240,7 @@ class PipelineController {
       )
     }
 
-    return front50Service.getPipelineConfigsForApplication((String) pipeline.get("application"), true)?.find {
+    return front50Service.getPipelineConfigsForApplication((String) pipeline.get("application"), null, true)?.find {
```

Comment:
There's another call to `front50Service.getPipelineConfigsForApplication` (e.g. in deletePipeline above) that also needs the extra pipelineNameFilter argument.

Reply:
That was the only place where the argument was missing and I fixed it now.